### PR TITLE
[ADD] sale_order_report_with_note (migrated sale_order_webkit to v8 in qweb and renamed the module)

### PR DESCRIPTION
**Pull request by serpentcs-dev1**

### CODE REVIEW

File: sale_order_report_with_note/README.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+* This module replaces the webkit Quotation / Sale Order report by
+  a qweb report and adds header and footer notes on the sale order 
+  and quotation
+
+
+Installation
+============
+
+No specific installation requirements.
```
Comment (LoisRForgeFlow) [inline sale_order_report_with_note/README.rst]: remove not used sections.

File: sale_order_report_with_note/__init__.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2011 Camptocamp SA (http://www.camptocamp.com)
+# @author Guewen Baconnier
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
```
Comment (LoisRForgeFlow) [inline sale_order_report_with_note/__init__.py]: One line copyrights

**Comment by LoisRForgeFlow** [comment]
@darshan-serpent Try to solve your conflicts, to make this mergeable.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
-[![Build Status](https://travis-ci.org/OCA/sale-reporting.svg?branch=8.0)](https://travis-ci.org/OCA/sale-reporting)
-[![Coverage Status](https://coveralls.io/repos/OCA/sale-reporting/badge.png?branch=8.0)](https://coveralls.io/r/OCA/sale-reporting?branch=8.0)
+[![Build Status](https://travis-ci.org/OCA/sale-reporting.svg?branch=9.0)](https://travis-ci.org/OCA/sale-reporting)
+[![Coverage Status](https://coveralls.io/repos/OCA/sale-reporting/badge.png?branch=9.0)](https://coveralls.io/r/OCA/sale-reporting?branch=9.0)
```
Comment (LoisRForgeFlow) [inline README.md]: Why do you change this to v9? this is still v8

Comment (serpentcs-dev1) [reply README.md]: Strange this is not done in the last commit, not sure when and why it was changed.

Comment (LoisRForgeFlow) [reply README.md]: yes, it's weird, it was added in the very first one (dfebab3d225b9eb0156afbcdeefa62059238dac3). Just revert the change  :scissors:

**Comment by rousseldenis** [comment]
@serpentcs-dev1 Could we consider this as too old and close ? @JayVora-SerpentCS

**Comment by JayVora-SerpentCS** [comment]
> @serpentcs-dev1 Could we consider this as too old and close ? @JayVora-SerpentCS

Please proceed yes!

**Pull request closed without merging**